documentation.html

```diff
@@ -272,6 +275,143 @@ <h5>Examples of Link, Float Button and Icon Buttons: </h5>
                 </iframe>
             </div>
             <!-- Button Component Ends  -->
+             <!-- Card Component Starts  -->
+             <div class="card-component">
+                <h3>Cards</h3>
+                <p>Cards are used to show the data which is related.Various types of cards: Basic Information Card,
+                    Horizontal, Vertical, Card with dismiss, Card with Text Overlay are showcased below with their code
+                    snippets.</p>
+                <div class="card-grid">
+                    <!-- Basic Card -->
+                    <div class="card card-basic">
+                        <img class="basic-img" src="https://material-components.github.io/material-components-web-catalog/static/media/photos/3x2/2.jpg"
+                            alt="" />
+                        <div class="basic-card-content">
+                            <h3>Our Changing Planet</h3>
+                            <p>by Kurt Wagner</p>
+                            <p>Visit ten places on our planet that are undergoing the biggest changes today.</p>
+                        </div>
+                        <div class="basic-card-elements">
+                            <a class="card-link" src="">READ</a>
+                            <a class="card-link" src="">BOOKMARK</a>
```

Comment:
a tag will contain href

Reply:
Made required changes, thanks for pointing it out